crates/sui-indexer-alt-graphql/src/api/types/snapshots/sui_indexer_alt_graphql__api__types__available_range__field_piplines_tests__registry_collect_pipelines_snapshot.snap

```diff
@@ -0,0 +1,2505 @@
+  => {"cp_sequence_numbers", "tx_digests"}
+
+Checkpoint.transactions (filter: kind)
+  => {"cp_sequence_numbers", "tx_digests"}
+
+Checkpoint.transactions (filter: affectedAddress)
+  => {"cp_sequence_numbers", "tx_digests"}
+
+Checkpoint.transactions (filter: affectedObject)
+  => {"cp_sequence_numbers", "tx_digests"}
+
+Checkpoint.transactions (filter: sentAddress)
+  => {"cp_sequence_numbers", "tx_digests"}
+
+CheckpointConnection.pageInfo
+  => {}
+
+CheckpointConnection.edges
+  => {}
+
+CheckpointConnection.nodes
+  => {}
+
+CheckpointEdge.node
+  => {}
+
+CheckpointEdge.cursor
+  => {}
+
+CoinDenyListStateCreateTransaction._
+  => {}
+
+CoinMetadata.address
+  => {}
+
+CoinMetadata.version
+  => {}
+
+CoinMetadata.digest
+  => {}
+
+CoinMetadata.balance
+  => {"consistent"}
+
+CoinMetadata.balances
+  => {"consistent"}
+
+CoinMetadata.contents
+  => {}
+
+CoinMetadata.decimals
+  => {}
+
+CoinMetadata.defaultSuinsName
+  => {}
+
+CoinMetadata.description
+  => {}
+
+CoinMetadata.dynamicField
+  => {"obj_versions"}
+
+CoinMetadata.dynamicFields
+  => {"consistent"}
+
+CoinMetadata.dynamicObjectField
+  => {"obj_versions"}
+
+CoinMetadata.hasPublicTransfer
+  => {}
+
+CoinMetadata.iconUrl
+  => {}
+
+CoinMetadata.multiGetDynamicFields
+  => {"obj_versions"}
+
+CoinMetadata.multiGetDynamicObjectFields
+  => {"obj_versions"}
+
+CoinMetadata.moveObjectBcs
+  => {}
+
+CoinMetadata.multiGetBalances
+  => {"consistent"}
+
+CoinMetadata.name
+  => {}
+
+CoinMetadata.objectAt
+  => {"obj_versions"}
+
+CoinMetadata.objectBcs
+  => {}
+
+CoinMetadata.objectVersionsAfter
+  => {"obj_versions"}
+
+CoinMetadata.objectVersionsAfter (filter: afterVersion)
+  => {"obj_versions"}
+
+CoinMetadata.objectVersionsAfter (filter: beforeVersion)
+  => {"obj_versions"}
+
+CoinMetadata.objectVersionsBefore
+  => {"obj_versions"}
+
+CoinMetadata.objectVersionsBefore (filter: afterVersion)
+  => {"obj_versions"}
+
+CoinMetadata.objectVersionsBefore (filter: beforeVersion)
+  => {"obj_versions"}
+
+CoinMetadata.objects
+  => {"consistent"}
+
+CoinMetadata.objects (filter: ownerKind)
+  => {"consistent"}
+
+CoinMetadata.objects (filter: owner)
+  => {"consistent"}
+
+CoinMetadata.objects (filter: type)
+  => {"consistent"}
+
+CoinMetadata.owner
+  => {}
+
+CoinMetadata.previousTransaction
+  => {}
+
+CoinMetadata.storageRebate
+  => {}
+
+CoinMetadata.receivedTransactions
+  => {"cp_sequence_numbers", "tx_affected_addresses", "tx_digests"}
+
+CoinMetadata.receivedTransactions (filter: afterCheckpoint)
+  => {"cp_sequence_numbers", "tx_affected_addresses", "tx_digests"}
+
+CoinMetadata.receivedTransactions (filter: atCheckpoint)
+  => {"cp_sequence_numbers", "tx_affected_addresses", "tx_digests"}
+
+CoinMetadata.receivedTransactions (filter: beforeCheckpoint)
+  => {"cp_sequence_numbers", "tx_affected_addresses", "tx_digests"}
+
+CoinMetadata.receivedTransactions (filter: function)
+  => {"cp_sequence_numbers", "tx_calls", "tx_digests"}
```

Comment:
There's seems to be an inconsistency here -- all the other `receivedTransactions` entries for `CoinMetadata` mention `tx_affected_addresses` except this one. In reality you can't make this call -- a function filter + an affected address filter would mean multiple filters, but probably it's more correct to say the retention comes from `tx_affected_addresses`?